src/styles/utils/sizing.ts

```diff
@@ -14,6 +14,10 @@ export default {
         height: 272,
     },
 
+    h52: {
+        height: 52,
+    },
+
```

Comment:
This does not belong here as it does not respect the base 4 rule (h52 should be 208)

Reply:
fixed